### PR TITLE
bugfix/WEBUMENIA-1344: sort collections in admin by creation date and…

### PR DESCRIPTION
**Pull request by martin-havala**
… allow to unpublish them

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (link to Jira or GitHub issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] I have updated the [CHANGELOG](../CHANGELOG.md)
- [x] I have requested at least 1 reviewer for this PR
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: app/Http/Controllers/CollectionController.php [hunk 1]
```diff
@@ -61,8 +61,15 @@ public function store()
...
+            if (Input::has('published_at')) {
+                $collection->published_at = Input::get('published_at');
+                if ($collection->published_at === '') {
+                    $collection->published_at = null;
+                }
+            } else {
+                $collection->published_at = null;
+            }
 
-            $collection->published_at = Input::get('published_at', null);
```
Comment (eronisko) [inline app/Http/Controllers/CollectionController.php:65]: This has me stumped... Doesn't Laravel do this out of the box?
https://laracasts.com/discuss/channels/laravel/best-way-to-convert-empty-string-from-input-to-null-for-db

Comment (martin-havala) [reply app/Http/Controllers/CollectionController.php:65]: is it enabled? i tried it with pure Input::get('published_at', null) and it was always stored as 0000-00-00 00:00:00

Comment (igor-kamil) [reply app/Http/Controllers/CollectionController.php:65]: check whether middleware `\Illuminate\Foundation\Http\Middleware\ConvertEmptyStringsToNull::class` is enabled

Comment (martin-havala) [reply app/Http/Controllers/CollectionController.php:65]: nope

Comment (martin-havala) [reply app/Http/Controllers/CollectionController.php:65]: and it doesn't seem it was enabled ever

Comment (igor-kamil) [reply app/Http/Controllers/CollectionController.php:65]: probably wasn't. would you add it?

Comment (martin-havala) [reply app/Http/Controllers/CollectionController.php:65]: I did

**Pull request merged**